### DISCUSSION TITLE
What host are we using?

### DISCUSSION THREAD

**Discussion by techpluscraft** [DevBase-Dev/twigbot#12] [General]
What host are we using to run the bot

**Comment by VetlixDev** [comment]
I got somehost.xyz which we can host on. If we want to use that. you guys gotta register. but I can keep it up

**Reply by VincentRPS** [reply]
What Is Samehost?

**Reply by VetlixDev** [reply]
its called somehost. Go to https://somehost.xyz/

**Reply by VincentRPS** [reply]
Yeah I can’t load that website lmao

**Reply by VetlixDev** [reply]
Vpn boyyy

**Reply by VincentRPS** [reply]
Seems good but mongo is better for database hosting that could be the bot host tho

**Reply by VetlixDev** [reply]
We are using somehost to host the bot and using mongo to  host data management

**Reply by VincentRPS** [reply]
Yes

**Comment by VincentRPS** [comment]
> What host are we using to run the bot

i Think We Should Use Mongodb The Servers Are Free Until We Use 500mbs Of Data My Code Has A Little Bit Of It Integrated Already Like I Imported Pymongo And Stuff, Plus Most Bots Use Them So There'll Be More Documentation For It

**Reply by VetlixDev** [reply]
Bruht there are better host which give 1,2 gb of data

**Reply by VincentRPS** [reply]
Yeah but discord integration is better an easier with mongo

**Reply by techpluscraft** [reply]
lets use mongo for db and somehost.xyz for host
